### PR TITLE
Coverity fixes for 1.3

### PR DESCRIPTION
**Pull request by cevian**

### CODE REVIEW

File: src/plan_expand_hypertable.c [hunk 1]
```diff
@@ -236,11 +236,13 @@ transform_time_bucket_comparison(PlannerInfo *root, OpExpr *op)
 				if (interval->month != 0)
 					return op;
 				if (DatumGetDateADT(castNode(Const, value)->constvalue) >=
-					DATEVAL_NOEND - interval->day + ceil(interval->time / USECS_PER_DAY))
+					DATEVAL_NOEND - interval->day +
+						ceil((double) interval->time / (double) USECS_PER_DAY))
```
Comment (JLockerman) [inline src/plan_expand_hypertable.c]: Has @svenklemm looked at this?

File: tsl/src/continuous_aggs/create.c [hunk 2]
```diff
@@ -519,8 +517,9 @@ get_timebucketfnoid()
 	Oid funcoid;
 	const char *funcname = TIMEBUCKETFN;
 	CatCList *catlist = SearchSysCacheList1(PROCNAMEARGSNSP, CStringGetDatum(funcname));
+	int i;
 
-	for (int i = 0; i < catlist->n_members; i++)
+	for (i = 0; i < catlist->n_members; i++)
```
Comment (RobAtticus) [inline tsl/src/continuous_aggs/create.c]: Can we figure out how to get the compiler to fail (either locally or on Travis) for this kind of thing? Sometimes it doesn't get caught and it causes problems when building on Debian/Ubuntu

Comment (cevian) [reply tsl/src/continuous_aggs/create.c]: Well  coverity fails but not noisily. I can ask @JLockerman to take a look. But not for this PR.

Comment (JLockerman) [reply tsl/src/continuous_aggs/create.c]: disable the warning everywhere 😜 
more seriously, the warning isn't widely available, I think it's only some versions of GCC that offer it.

File: src/plan_expand_hypertable.c (hunk 1 above)
Comment (svenklemm) [reply src/plan_expand_hypertable.c]: LGTM

**Pull request merged**